src/libasr/codegen/llvm_utils.cpp

```diff
@@ -1267,6 +1267,7 @@ namespace LCompilers {
                 break;
             }
             case (ASR::ttypeType::CPtr) : {
+                a_kind = 8;
```

Comment:
Previously, we used `a_kind = 4`, but now I set `8` to be default, this is required for Symbolic as we use `i64` pointer for `CPtr`.

Reply:
I thought CPtr doesn't have a kind?

Reply:
How do we decide the malloc size for `list[CPtr]`?

Reply:
I went through the changes and the changes do work in the correct direction. If we could know whether `a_kind` would be required or not for `list[CPtr]`, the PR can be finished accordingly .

Reply:
cc @certik 
Could you clarify Thirumalai's comment here? I think this is close to completion once we have an idea for the same.